projects/js-packages/publicize-components/src/components/share-buttons/share-buttons.tsx

```diff
@@ -1,34 +1,11 @@
 import { SocialServiceIcon } from '@automattic/jetpack-components';
+import { useAnalytics } from '@automattic/jetpack-shared-extension-utils';
 import { Button } from '@wordpress/components';
-import React from 'react';
+import React, { useCallback } from 'react';
```

Comment:
This should come from `wordpress/elements` although I'm not sure that exposes the types. I think you can `import type` instead for that.

Reply:
We have imports from `react` everywhere and yes it doesn't import types. Since React is still loaded externally, using `import type` doesn't make any difference here.

Reply:
Updated. Thanks

Reply:
May be there should be ESLint rules to prevent non `type` imports from `react`.